docs/src/development/local/lando.md

```diff
@@ -1,46 +1,124 @@
+To use a build hook that requires environment variables, manually [add them](https://docs.lando.dev/platformsh/config.html#environment-variables).
 
-```bash
-$ lando init --source cwd --recipe platformsh
-```
+## 1. Install Lando
 
-That will use your existing git clone and add the appropriate Lando configuration.
+Follow the [Lando installation instructions](https://docs.lando.dev/getting-started/installation.html).
 
-Either way, start the local environment by running:
+## 2. Initialize Lando
 
-```bash
-$ lando start
-```
+{{< codetabs >}}
 
-You can then download data from the environment for your current branch and load it into your local Lando environment by running:
+---
+title=On a new Platform.sh project without code
+file=none
+highlight=false
+---
+
+For a quicker start, create a project based on the Platform.sh [PHP template](https://github.com/platformsh-templates/php).
+The template provides the most basic configuration for running a custom PHP project built with Composer.
+It also includes the required Platform.sh configuration files out of the box.
+
+1. [Create a new project based on the PHP template](https://console.platform.sh/projects/create-project/template?query=php). <!-- TODO: The link probably has to change to have a specific source and campaign for everything `utm` -->
+2. Clone that project locally with Platform.sh [CLI](../../gettingstarted/introduction/own-code/cli-install.md) `platform get <PROJECT_ID>` or with [git](../../administration/web/_index.md#git).
+3. In the project's folder, run `lando init --recipe platformsh --source platformsh`.
+4. Follow the instructions provided by the interactive prompt.
+On the `Which project?` step, select the project created in the first step.
+
+<--->
+
+---
+title=On an existing Platform.sh project
+file=none
+highlight=false
+---
+
+If your code isn't present locally, retrieve your codebase with one of these methods:
+
+- Using the [Platform.sh CLI](../../gettingstarted/introduction/own-code/cli-install.md) by running `platform get <PROJECT_ID>`
+- Using [git](../../administration/web/_index.md#git)
+
+Otherwise, access the directory with your project.
+
+Run `lando init --recipe platformsh --source cwd` and follow the instructions provided by the interactive prompt.
+
+{{< /codetabs >}}
+
+The `init` command generates the `.lando.yml` file required to start Lando.
+
+## 3. Start Lando
+
+To start your app and services, run `lando start`.
+
+## 4. Access your local app
+
+The last lines of the previously ran `lando start` command contains URL to the different app and services.
+Access your app and services by opening the according URLs in your browser.
+
+## What's next
+
+- [Import data and download files](https://docs.lando.dev/platformsh/sync.html) from your remote Platform.sh site.
+- If you made changes in the Platform.sh [configuration files](../../overview/structure.md) during development, run `lando rebuild` for these to be taken into account in Lando.
```

Comment:
:hammer_and_wrench: Because it's about next steps and what might happen in the future, this is possible
```suggestion
- If you make changes in the Platform.sh [configuration files](../../overview/structure.md) during development, run `lando rebuild` for these to be taken into account in Lando.
```

Reply:
Sounds better indeed  :)